mitodl/micromasters
# Added table block in benefits page content

#### Pre-Flight checklist




#### What are the relevant tickets?
Fixes #4664 

#### What's this PR do?
This PR adds support of table blocks on the benefit page's content.

#### How should this be manually tested?
1. Open `/cms` on micromasters and create/update the benefits page. You should be able to see/add the Benefits table option inside benefits page content.
2. Open `/benefits` page on micromasters instance. You should see the added tables here.


#### Screenshots (if appropriate)

**(Benefits table option in cms)**

<img width="1241" alt="Screenshot 2020-10-26 at 3 31 09 PM" src="https://user-images.githubusercontent.com/34372316/97163828-0707ec80-17a3-11eb-8e45-aaceb028544e.png">

**(Benefits table creation in cms)**

<img width="1205" alt="Screenshot 2020-10-26 at 3 34 21 PM" src="https://user-images.githubusercontent.com/34372316/97163838-0a02dd00-17a3-11eb-8e0c-dc7f1b086094.png">

**(Created benefits table display on Micromasters)**

<img width="1434" alt="Screenshot 2020-10-26 at 3 41 32 PM" src="https://user-images.githubusercontent.com/34372316/97163840-0b340a00-17a3-11eb-9a40-5859c6ac8116.png">

**(Mobile screenshot)**
<img width="414" alt="Screenshot 2020-10-26 at 3 56 00 PM" src="https://user-images.githubusercontent.com/34372316/97164416-e8eebc00-17a3-11eb-8ae1-3844569d6567.png">
